app/install/dcos.md

```diff
@@ -75,8 +76,9 @@ have basic knowledge of [DC/OS]({{ page.links.dcos }}),
 
 5. **Deploy Kong**
 
-    Using the `kong_<postgres|cassandra>.json` file from the
-    kong-dist-mesos repo, deploy Kong in the cluster:
+    Using the `kong_<postgres|cassandra>.json` file from the kong-dist-dcos
```

Comment:
That how its was published 1st and reviewed by @thibaultcha. I don't backticks needes here